### PR TITLE
[CHNAGE][LP-1686] record country to `UserProfile` model.

### PR DESCRIPTION
**Pull request by qasimgulzar**
**Acceptance Criteria:**
1. The country should be start recoding in `UserProfile` model.

### CODE REVIEW

File: openedx/features/partners/give2asia/forms.py [hunk 1]
```diff
@@ -13,6 +13,8 @@
 from student.models import CourseEnrollmentAllowed, email_exists_or_retired
 from util.password_policy_validators import validate_password
 
+from lms.djangoapps.onboarding.helpers import   COUNTRIES
```
Comment (zahid-arbisoft) [inline openedx/features/partners/give2asia/forms.py]: Lets remove extra space before `COUNTRIES`

File: openedx/features/partners/give2asia/forms.py [hunk 2]
```diff
@@ -100,6 +102,13 @@ def clean_registration_data(self, registration_data):
...
         registration_data.update(self.cleaned_data)
 
+    def clean_country(self):
+        country = self.cleaned_data['country']
+
+        cleaned_country=next((code for code,label in COUNTRIES.items() if label.lower()==country.lower()),None)
+        if cleaned_country:
+            return cleaned_country
+        raise forms.ValidationError(ugettext_noop('Please select country.'))
     username = UsernameField()
```
Comment (zahid-arbisoft) [inline openedx/features/partners/give2asia/forms.py]: Lets have one line space after `clean_country` function

File: openedx/features/partners/give2asia/forms.py [hunk 3]
```diff
@@ -100,6 +102,13 @@ def clean_registration_data(self, registration_data):
         """Once form is validated, call this function to get clean registration data"""
         registration_data.update(self.cleaned_data)
 
+    def clean_country(self):
+        country = self.cleaned_data['country']
+
+        cleaned_country=next((code for code,label in COUNTRIES.items() if label.lower()==country.lower()),None)
```
Comment (zahid-arbisoft) [inline openedx/features/partners/give2asia/forms.py]: Lets have proper spacing in this line

File: openedx/features/partners/give2asia/forms.py [hunk 4]
```diff
@@ -125,3 +134,9 @@ def clean_registration_data(self, registration_data):
             "required": "Organization name is required",
         },
     )
+
+    country = forms.CharField(
+        error_messages={
+            "required": "Country name is required",
```
Comment (zeshanarifios) [inline openedx/features/partners/give2asia/forms.py]: Make it translation string

File: openedx/features/partners/give2asia/forms.py [hunk 5]
```diff
@@ -100,6 +102,13 @@ def clean_registration_data(self, registration_data):
         """Once form is validated, call this function to get clean registration data"""
         registration_data.update(self.cleaned_data)
 
+    def clean_country(self):
+        country = self.cleaned_data['country']
+
+        cleaned_country=next((code for code,label in COUNTRIES.items() if label.lower()==country.lower()),None)
+        if cleaned_country:
+            return cleaned_country
+        raise forms.ValidationError(ugettext_noop('Please select country.'))
```
Comment (zeshanarifios) [inline openedx/features/partners/give2asia/forms.py]: How about using ValidationError

File: openedx/features/partners/give2asia/forms.py [hunk 6]
```diff
@@ -13,6 +13,8 @@
 from student.models import CourseEnrollmentAllowed, email_exists_or_retired
 from util.password_policy_validators import validate_password
 
+from lms.djangoapps.onboarding.helpers import   COUNTRIES
+from django.utils.translation import ugettext_noop
```
Comment (zeshanarifios) [inline openedx/features/partners/give2asia/forms.py]: make it consistent as we are using lazytext

File: openedx/features/partners/give2asia/forms.py (hunk 4 above)
Comment (zeshanarifios) [reply openedx/features/partners/give2asia/forms.py]: use single quotes to make it consistent

**Pull request merged**